Sources/Atomic.swift

```diff
@@ -140,34 +136,31 @@ internal class Lock: LockProtocol {
 	}
 	#endif
 
-	#if compiler(>=5.7)
-		#if !targetEnvironment(macCatalyst)
```

Comment:
Now that macOS 13 is released, we can remove this silly macCatalyst dance and simplify the conditions a lot.